tools/clang/lib/CodeGen/CGHLSLMS.cpp

```diff
@@ -1005,9 +1010,9 @@ unsigned CGMSHLSLRuntime::ConstructStructAnnotation(DxilStructAnnotation *annota
         fieldSemName = sd->SemanticName;
       } break;
       case hlsl::UnusualAnnotation::UA_ConstantPacking: {
-        const hlsl::ConstantPacking *cp = cast<hlsl::ConstantPacking>(it);
-        CBufferOffset = cp->Subcomponent << 2;
-        CBufferOffset += cp->ComponentOffset;
+        packOffset = cast<hlsl::ConstantPacking>(it);
+        CBufferOffset = packOffset->Subcomponent << 2;
+        CBufferOffset += packOffset->ComponentOffset;
```

Comment:
I'm not sure if we hit this code actually, since pack offset appears to normally be handled elsewhere, and doesn't appear to be allowed on struct fields.  I wasn't absolutely sure, so didn't delete the code yet.